.gitignore

```diff
@@ -19,3 +19,6 @@ build/
 
 # OSX
 .DS_Store
+dist/public/js/react-d3.min.js
+dist/public/js/react-d3.min.js.map
+test.js
```

Comment:
what's `test.js` for. If you wish to ignore these file I think it's better to do it in another pr.